spec/support/api/schemas/payments.json

```diff
@@ -0,0 +1,35 @@
+{
+  "type": "array",
+  "required": ["payments"],
+  "items": {
+    "type": "object",
+    "required": [
+      "agency_id",
+      "amount_cents",
+      "care_finished_on",
+      "care_started_on",
+      "created_at",
+      "discrepancy_cents",
+      "id",
+      "paid_on",
+      "site_id",
+      "slug",
+      "updated_at"
+    ],
+    "properties": {
+      "agency_id": { "type": "string", "format": "uuid" },
+      "amount_cents": { "type": "integer", "minimum": 0 },
+      "amount_currency": { "type": "string" },
+      "care_finished_on": { "type": "string", "format": "date" },
+      "care_started_on": { "type": "string", "format": "date" },
+      "created_at": { "type": "string" , "format": "date-time" },
+      "discrepancy_cents": { "type": "integer", "minimum": 0 },
```

Comment:
Does "minimum" imply it has to have a discrepancy?  There's a difference between a discrepancy not being recorded, and it being 0.  I'd like to retain the `null` state as an option, and I wonder if defining this as a minimum means there has to be a value.